src/app/core/facades/shopping.facade.ts

```diff
@@ -222,4 +223,10 @@ export class ShoppingFacade {
     });
     return this.store.pipe(select(getPromotions(promotionIds)));
   }
+
+  // REVIEWS
+  getProductReviews$(sku: string) {
+    this.store.dispatch(loadProductReviews({ sku }));
```

Comment:
I suggest to check here if the feature is enabled - so an action LoadProductReviews is never called if the feature is off:
 getProductReviews$(sku: string) {
    if (this.featureToggleService.enabled('rating')) {
      this.store.dispatch(loadProductReviews({ sku }));
    }
    return this.store.pipe(select(getProductReviewsBySku(sku)));
  }